include/triton/core/tritonserver.h

```diff
@@ -1692,16 +1692,39 @@ TRITONSERVER_DECLSPEC TRITONSERVER_Error*
 TRITONSERVER_ServerOptionsSetCudaMemoryPoolByteSize(
     TRITONSERVER_ServerOptions* options, int gpu_device, uint64_t size);
 
-/// Set the total response cache byte size that the server can allocate in CPU
-/// memory. The response cache will be shared across all inference requests and
-/// across all models.
+// Deprecated. See TRITONSERVER_ServerOptionsSetCacheConfig instead.
+TRITONSERVER_DECLSPEC TRITONSERVER_Error*
```

Comment:
Can you add the old comment based on the discussion here:

https://github.com/triton-inference-server/core/pull/142/files#r1046033041